static/app/utils/theme.tsx

```diff
@@ -794,19 +1071,20 @@ const commonTheme = {
   headerSelectorLabelHeight: 28,
 
   // Relative font sizes
+  // @TODO(jonasbadalic) why do we need these
   fontSizeRelativeSmall: '0.9em',
-
-  fontSizeExtraSmall: '11px' as const,
-  fontSizeSmall: '12px' as const,
-  fontSizeMedium: '14px' as const,
-  fontSizeLarge: '16px' as const,
-  fontSizeExtraLarge: '18px' as const,
-  codeFontSize: '13px' as const,
-  headerFontSize: '22px' as const,
-
-  fontWeightNormal: 400 as const,
-  fontWeightBold: 600 as const,
-
+  fontSizeExtraSmall: '11px',
+  fontSizeSmall: '12px',
+  fontSizeMedium: '14px',
+  fontSizeLarge: '16px',
+  fontSizeExtraLarge: '18px',
+  codeFontSize: '13px',
+  headerFontSize: '22px',
+
+  fontWeightNormal: 400,
+  fontWeightBold: 600,
```

Comment:
Do we know why the `as const` were added here? Maybe for completions somewhere?

Reply:
I have no idea, but completion should still work, the shape still gets inferred